core/templates/dev/head/pages/teach.html

```diff
@@ -48,56 +48,51 @@
-                your creations with the community.
-              </p>
-            </div>
-            <div class="oppia-about-three-cols-layout oppia-about-three-cols-feature">
-              <img ng-src="<[getStaticImageUrl('/general/creator-feedback.png')]>">
-              <h3>Earn feedback</h3>
-              <p style="margin-top: -10px;">
-                to improve your exploration.
-              </p>
-            </div>
+          <div style="text-align: center; width: 100%;">
+            <em>Create high-quality lessons that help students learn difficult topics.</em>
           </div>
 
-          <hr>
+          <p>
+            You might know a student who's struggling with a topic at school.
+            They really want to learn it, but the lesson goes too fast, or the
+            textbook isn't easy to understand &mdash; and it's hard for them to
+            get the support they need. And it's not just this student: other
+            students, all over the world, are likely to struggle with the same
+            topics, again and again and again and again.
+          </p>
 
           <p>
-            How do pendulums work? Is light a particle or a wave? What is art?
-            Can you explain topics like these to a student who wants to learn
-            more? If so, you can use Oppia's unique technology to recreate 1-on-1
-            tutoring and reach students around the world by creating and sharing
-            a single piece of learning material.
+            In order to help address this, we're excited to announce the
+            <em>Teach with Oppia</em> program!
           </p>
 
           <p>
-            With Oppia, you can create <em>explorations</em> that help students
-            learn in a fun and creative way, using videos, images, and open-ended
-            questions. Students often have similar misconceptions, and Oppia
-            helps you organize these so that you can give targeted feedback to a
-            group of students at a time. By teaching with Oppia, you can
-            communicate with students around the world, understand different
-            points of view, and improve your skills in communication and empathy.
+            The <em>Teach with Oppia</em> program enables <strong>anyone</strong>
+            &mdash; especially students &mdash; to contribute to a collection
+            of free, accessible lessons on topics that learners find difficult.
+            The lessons use Oppia's unique technology to recreate 1-on-1
+            tutoring, and typically include clear, compelling and motivated
+            explanations, together with ample exercises that allow students to
+            gain proficiency with the subject. Students can then do these
+            lessons at their own pace, supported as much as possible by useful,
+            targeted feedback.
           </p>
+
+          <p>
+            Currently, participation in the program is <strong>free</strong>.
```

Comment:
This is complicated. My main aim was simply to be clear that there's no guarantee that we'll be able to offer this program at no cost indefinitely, since it consumes Foundation resources. So I think the "currently" is important, but I do see what you mean.

I've thought about the trade-off at length, and I think I'm going to keep the wording as-is for now and submit this PR. However, I'll also start a separate discussion thread to discuss this in more detail, and will make any necessary changes before the next release. Thanks for the feedback!